collectors/idlejitter.plugin/metadata.yaml

```diff
@@ -2,31 +2,38 @@ meta:
   plugin_name: idlejitter.plugin
   module_name: idlejitter.plugin
   monitored_instance:
-    name: IdleJitter
+    name: Idle OS Jitter
     link: ''
     categories:
-    - data-collection.synthetic-checks
+      - data-collection.synthetic-checks
     icon_filename: 'syslog.png'
   related_resources:
     integrations:
       list: []
   info_provided_to_referring_integrations:
     description: ''
-  keywords: []
+  keywords:
+    - latency
+    - jitter
   most_popular: false
 overview:
   data_collection:
-    metrics_description: 'Monitor IdleJitter performance for optimal system idle process operations. Monitor CPU idle times, wake-ups, and power states to optimize system resource usage.'
-    method_description: ''
+    metrics_description: >
```

Comment:
Should the > be a | ?

Reply:
`>` is used when you want newlines to be rendered as spaces, essentially you can break the string into multiple lines and it will be a single line in the end.

Reply:
[YAML 
Multiline](https://yaml-multiline.info/)
in deta
ils 😄

Reply:
It’s intentionally a flow block style here, because folding for this stuff _should_ be handled by the markdown processor or the HTML processing done by the user’s browser, not here in the YAML file.